recipe/conda_build_config.yaml

```diff
@@ -1,24 +1,18 @@
 cross_target_platform:
-  - linux-64                            # [linux]
-  - linux-aarch64                       # [linux]
-  - linux-ppc64le                       # [linux]
-  - osx-64                              # [linux or (osx and x86_64) or (osx and arm64)]
-  - osx-arm64                           # [linux or (osx and x86_64) or (osx and arm64)]
-  - win-64                              # [linux64 or win]
+  - linux-64                            # [linux and x86_64]
```

Comment:
```suggestion
  - linux-64                            # [linux64]
```

Reply:
My apologies for reviewing this during a hectic time and suggesting otherwise. I do think `linux64` is sufficient here based on [this](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors).